src/components/__tests__/row.test.js

```diff
@@ -13,30 +13,36 @@ describe('<Row />', () => {
   it('should have a reverse direction for each media', () => {
     const tree = renderer.create(<Row reverse />).toJSON()
 
-    expect(tree).toHaveStyleRule(
-      'flex-direction', `row-reverse`)
+    expect(tree).toHaveStyleRule('flex-direction', `row-reverse`)
+    expect(tree).toHaveStyleRule('flex-wrap', `wrap-reverse`)
   })
 
   it('should have a reverse direction for each media', () => {
     const tree = renderer.create(<Row reverse />).toJSON()
 
-    expect(tree).toHaveStyleRule(
-      'flex-direction', `row-reverse`)
+    expect(tree).toHaveStyleRule('flex-direction', `row-reverse`)
+    expect(tree).toHaveStyleRule('flex-wrap', `wrap-reverse`)
   })
 
   it('should have a reverse direction for specific screen', () => {
     const tree = renderer.create(<Row reverse={['md', 'lg']} />).toJSON()
     const { breakpoints } = BASE_CONF
 
-    expect(tree).toHaveStyleRule(
-      'flex-direction', `row-reverse`, {
-        media: `only screen and (min-width: ${breakpoints['md']}rem)`
-      })
+    expect(tree).toHaveStyleRule('flex-direction', `row-reverse`, {
```

Comment:
Here, I believe it is useful to test the other dimensions to ensure that we are applying the reverse flow only to the dimensions that have been informed

```suggestion
        expect(tree).toHaveStyleRule('flex-direction', 'row', {
      media: `only screen and (min-width: ${breakpoints.xs}rem)`
    })

    expect(tree).toHaveStyleRule('flex-wrap', undefined, {
      media: `only screen and (min-width: ${breakpoints.xs}rem)`
    })

    expect(tree).toHaveStyleRule('flex-direction', 'row', {
      media: `only screen and (min-width: ${breakpoints.sm}rem)`
    })

    expect(tree).toHaveStyleRule('flex-wrap', undefined, {
      media: `only screen and (min-width: ${breakpoints.sm}rem)`
    })

    expect(tree).toHaveStyleRule('flex-direction', 'row-reverse', {
      media: `only screen and (min-width: ${breakpoints.md}rem)`
    })

    expect(tree).toHaveStyleRule('flex-wrap', 'wrap-reverse', {
      media: `only screen and (min-width: ${breakpoints.md}rem)`
    })

    expect(tree).toHaveStyleRule('flex-direction', 'row-reverse', {
      media: `only screen and (min-width: ${breakpoints.lg}rem)`
    })

    expect(tree).toHaveStyleRule('flex-wrap', 'wrap-reverse', {
      media: `only screen and (min-width: ${breakpoints.lg}rem)`
    })

    expect(tree).toHaveStyleRule('flex-direction', 'row', {
      media: `only screen and (min-width: ${breakpoints.xl}rem)`
    })

    expect(tree).toHaveStyleRule('flex-wrap', undefined, {
      media: `only screen and (min-width: ${breakpoints.xl}rem)`
    })
```

Reply:
👍🏼